libminifi/include/RemoteProcessorGroupPort.h

```diff
@@ -81,7 +81,7 @@ class RemoteProcessorGroupPort : public core::Processor {
   /*!
    * Create a new processor
    */
-  RemoteProcessorGroupPort(const std::shared_ptr<io::StreamFactory> &stream_factory, std::string name, std::string url, const std::shared_ptr<Configure> &configure, utils::Identifier uuid = utils::Identifier()) // NOLINT
+  RemoteProcessorGroupPort(const std::shared_ptr<io::StreamFactory> &stream_factory, const std::string& name, std::string url, const std::shared_ptr<Configure> &configure, const utils::Identifier& uuid = {}) // NOLINT
```

Comment:
one more inconsistency here

Reply:
Fixed in a8574d103f96566edec377a0d7e1447e33ef14b9